modules/microshift-deploying-a-load-balancer.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-deploying-a-load-balancer_{context}"]
+= Deploying a TCP load balancer on a workload
+
+{product-title} offers a built-in implementation of network load balancers. This process uses the node IP address as the external IP address for the `LoadBalancer` service config file. 
+
+.Prerequisites 
+
+* You installed the OpenShift CLI (`oc`)
+* Access to the cluster as a user with the cluster-admin role.
+* A cluster installed on infrastructure configured with the OVN-Kubernetes network plugin.
+* The KUBECONFIG environment variable is set.
+
+.Procedure 
+
+. Verify that your pods are running with the following command:
++
+[source,terminal]
+----
+$ oc get pods -A
+----
+
+. Create a namespace with the following commands. This deploys a test application that verifies the connection.
++ 
+[source,terminal]
+----
+$ NAMESPACE=nginx-lb-test
+----
++
+[source,terminal]
+----
+$ oc create ns $NAMESPACE
+----
++
+[source,terminal]
+----
+$ oc adm policy add-scc-to-user privileged -z default -n $NAMESPACE
+----
+
+. Deploy three replicas of the test `nginx` application in your namespace.
++
+[source,terminal]
+----
+$ oc apply -n $NAMESPACE -f https://raw.githubusercontent.com/openshift/microshift/main/docs/config/nginx-IP-header.yaml
+----
+
+. You can verify that the three replicas started successfully by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n $NAMESPACE
+----
+
+. Create a `LoadBalencer` service config file for the `nginx` test application with the following commands:
++
+[source,terminal]
+----
+$ oc create -n $NAMESPACE -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
+spec:
+  ports:
+  - port: 81
+    targetPort: 8080
+  selector:
+    app: nginx
+  type: LoadBalancer
+EOF
+----
++
+[NOTE]
+====
+Do not use a port that is already occupied by the host process. 
+====
+
+
+. To verify that the service file exists and the external IP address is properly assigned, and the external IP is identical to the node IP, run the following command:
++
+[source,terminal]
+----
+$ oc get svc -n $NAMESPACE
+----
++
+.Example output 
+[source,terminal]
+----
+NAME    TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)        AGE
+nginx   LoadBalancer   10.43.183.104   192.168.1.241   80:32434/TCP   2m
```

Comment:
It should be '81:32434/TCP', as we use `port:81` above.

Reply:
Oh goodness, right, good eye. This is when you know its getting to the end of the week 😆